### PR TITLE
Split the diagnostic into new CAC002 for `ConfigureAwait(true)`.

### PR DESCRIPTION
**Pull request by cincuranet**

### CODE REVIEW

**Comment by cincuranet** [comment]
Better design (using `enum`) for #18.

File: ConfigureAwaitChecker.Analyzer/DiagnosticAnalyzer.cs [hunk 1]
```diff
@@ -10,15 +10,16 @@ namespace ConfigureAwaitChecker.Analyzer
...
-		static readonly string MessageFormat = "Possibly missing `ConfigureAwait(false)` call";
+		const string Title = "ConfigureAwaitChecker";
 		const string Category = "Code";
 
-		static readonly DiagnosticDescriptor Rule = new DiagnosticDescriptor(DiagnosticId, Title, MessageFormat, Category, DiagnosticSeverity.Warning, isEnabledByDefault: true);
+		public const string MissingConfigureAwaitFalseId = "CAC001";
+		public const string ConfigureAwaitWithTrueId = "CAC002";
+
+		static readonly DiagnosticDescriptor MissingConfigureAwaitFalseRule = new DiagnosticDescriptor(MissingConfigureAwaitFalseId, Title, "Possibly missing `ConfigureAwait(false)` call", Category, DiagnosticSeverity.Warning, isEnabledByDefault: true);
+		static readonly DiagnosticDescriptor ConfigureAwaitWithTrueRule = new DiagnosticDescriptor(MissingConfigureAwaitFalseId, Title, "Possibly wrong `ConfigureAwait(true)` call", Category, DiagnosticSeverity.Warning, isEnabledByDefault: true);
```
Comment (pmiossec) [inline ConfigureAwaitChecker.Analyzer/DiagnosticAnalyzer.cs]:
```suggestion
		static readonly DiagnosticDescriptor ConfigureAwaitWithTrueRule = new DiagnosticDescriptor(ConfigureAwaitWithTrueId, Title, "Possibly wrong `ConfigureAwait(true)` call", Category, DiagnosticSeverity.Warning, isEnabledByDefault: true);
```

**Review by pmiossec — CHANGES_REQUESTED** [review]
Except this little fix, that works great.
I'm waiting for the release now 😉

**Comment by cincuranet** [comment]
Thanks. There goes copy-and-paste. :)

**Comment by cincuranet** [comment]
And it's [released](https://www.nuget.org/packages/ConfigureAwaitChecker.Analyzer/3.0.0).

**Comment by pmiossec** [comment]
🎆 👍

**Pull request closed without merging**